lib/components/ServerTest.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from "react";
+import { io, Socket } from "socket.io-client";
+
+let socket: Socket;
+const ServerTest = () => {
```

Comment:
So this file is a basic boilerplate for me to fiddle with and experiment right?

Reply:
This is an example component I made that showcases how the front end can interact with the backend code. You can use it as a reference to build components that can interact with it. My bad on the lack of function description.